invenio_drafts_resources/services/records/service.py

```diff
@@ -181,19 +182,40 @@ def publish(self, id_, identity, links_config=None):
         """Publish a draft."""
         self.require_permission(identity, "publish")
 
-        # Get draft
+        # Get data layer draft
         draft = self.draft_cls.pid.resolve(id_, registered_only=False)
-
+        # At this point, draft has
+        # {..., "access": {'record': 'public', 'files': 'public', 'owned_by': [{'user': 1}]}}
+        print("REcordDraftService - publish - draft['access']", draft["access"])
```

Comment:
So far so good, we have a data layer representation that has "access" with the right values.